Read_Sorter_FASTQ.py

```diff
@@ -0,0 +1,78 @@
+class Read:
+    def __init__(self, read_id = 'ngs_read',
+                 read_sequence = 'read',
+                 comment = '+', quality = 'qual'):

```

Comment:
Не думаю, что этому классу вообще нужны значения по-умолчанию, тем более такие как "read" в качестве последовательности
```suggestion
    def __init__(self, read_id='ngs_read',
                 read_sequence='read',
                 comment='+', quality='qual'):

```